charts/redpanda/values.yaml

```diff
@@ -236,6 +236,25 @@ storage:
   # If not specified, then `emptyDir` will be used instead.
   # If specified, but `persistentVolume.enabled` is `true`, then has no effect.
   hostPath: ""
+  # Absolute path on host to store Redpanda's tiered storage cache data.
+  # If not specified, then `emptyDir` will be used instead.
+  # If specified, but `tieredStoragePersistentVolume.enabled` is `true`, then has no effect.
+  tieredStorageHostPath: ""
+  # If `enabled` is `true` then a PersistentVolumeClaim will be created for tiered storage cache and
+  # used to store data retrieved from cloud storage (eg. S3), otherwise `hostPath` is used.
+  tieredStoragePersistentVolume:
```

Comment:
We have the `storage.tieredStoragePersistentVolume`, and then we also have `storage.persistentVolume.tieredConfig`. Would it not make sense to move all `storage.persistentVolume.tieredConfig` into `storage.tieredStoragePersistentVolume`?

Reply:
In this PR there is no `storage.persistentVolume.tieredConfig`, but rather `storage.tieredConfig`. The `storage.tieredConfig` is representation of Redpanda tiered storage configuration. If we move `storage.tieredConfig` under `storage.tieredStoragePersistentVolume` then it would be miss leading as you could create ephemeral tiered storage cache volume.